docs/resources/aws_elasticloadbalancingv2_listener_certificates.md

```diff
@@ -0,0 +1,66 @@
+---
+title: About the aws_elasticloadbalancingv2_listener_certificates Resource
+platform: aws
+---
+
+# aws\_elasticloadbalancingv2\_listener\_certificates
+
+Use the `aws_elasticloadbalancingv2_listener_certificates` InSpec audit resource to test properties of a plural AWS ElasticLoadBalancingV2 Listener Certificates.
+
+## Syntax
+
+Ensure that a listener_arn exists.
+    describe aws_elasticloadbalancingv2_listener_certificates(listener_arn: 'test1') do
+      it { should exist }
+    end
+
+## Parameters
+
+##### listener_arn _(required)_
+
+See also the [AWS documentation on ElasticLoadBalancingV2 Listener Certificate](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-elasticloadbalancingv2-listenercertificate.html).
+
+## Properties
+
+|Property | Description|
+| --- | --- |
+| certificate_arn | The Amazon Resource Name (ARN) of the certificate. |
+| is_default | Indicates whether the certificate is the default certificate. Do not set this value when specifying a certificate as an input. This value is not included in the output when describing a listener, but is included when describing listener certificates. |
+
+## Examples
+
+### Ensure a listener_arn is available.
+    describe aws_elasticloadbalancingv2_listener_certificates(listener_arn: 'test1') do
+      it { should exist }
+    end
+
+
+##### Ensure that listerner has a desired certificate arn attached.
+    describe aws_elasticloadbalancingv2_listener_certificates(listener_arn: 'test1') do
+      its('certificate_arns') { should eq "test" }
```

Comment:
Is this going to return multiple certificate arns? 
Should this be `its('certificate_arns') { should include "test" }`